src/udev/udev-builtin-blkid.c

```diff
@@ -212,6 +212,9 @@ static int find_gpt_root(UdevEvent *event, blkid_probe pr, const char *loop_back
                         if (flags & SD_GPT_FLAG_NO_AUTO)
                                 continue;
 
+                        if (streq_ptr(label, "_empty"))
```

Comment:
Maybe, better to add a short comment about that here, not only in commit message.